I think BHPC needs to also be run, at least locally, since main seems to not be in the best of shapes. For an unrelated change I got
https://github.com/tenstorrent/tt-metal/actions/runs/24261161716/job/70845944802#step:7:13895
```
FAILED tests/ttnn/unit_tests/operations/fused/test_group_norm.py::test_sdxl_base_group_norm_bh[device_params={'l1_small_size': 0}-input_shape=(1, 512, 128, 128)] - AssertionError: Relative Frobenius norm of error 0.033935546875 exceeds threshold 0.03.
```
e.g.
```
pytest --timeout 300 tests/ttnn/unit_tests/operations/fused
```

--- a/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
+++ b/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
@@ -1296,7 +1296,7 @@ def test_group_norm_optional_weight_bias(device, N, C, H, W, num_groups, use_wel
     tt_output = ttnn.to_torch(tt_output)
 
     if use_welford:
-        pcc_threshold = 0.999
+        pcc_threshold = 0.99
         rtol = 0.14
         atol = 0.3
         frobenius_threshold = 0.06

--- a/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
+++ b/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
@@ -694,7 +694,7 @@ def test_sdxl_base_group_norm_bh(device, input_shape, perf_test_mode=False):
         pcc_threshold = 0.9999
         rtol = 0.065
         atol = 0.065
-        frobenius_threshold = 0.03
+        frobenius_threshold = 0.036
         assert_numeric_metrics(
             torch_output_tensor,
             tt_output_tensor,

--- a/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
+++ b/tests/ttnn/unit_tests/operations/fused/test_group_norm.py
@@ -1296,10 +1296,10 @@ def test_group_norm_optional_weight_bias(device, N, C, H, W, num_groups, use_wel
     tt_output = ttnn.to_torch(tt_output)
 
     if use_welford:
-        pcc_threshold = 0.99975
+        pcc_threshold = 0.999
         rtol = 0.14
-        atol = 0.085
-        frobenius_threshold = 0.02
+        atol = 0.3
+        frobenius_threshold = 0.06
     else:
         pcc_threshold = 0.9999
         rtol = 0.065

